### PR TITLE
feat: add --ipv4 & --ipv6 flags

### PR DESCRIPTION
**Pull request by radulucut**
#105 

Adds support for `--ipv4` and `--ipv6` flags.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +105,8 @@ Flags:
                        - an ID of a previous measurement to run with its probes
                        (default "world")
   -h, --help          help for globalping
+  -4, --ipv4          resolve names to IPv4 addresses
```
Comment (jimaek) [inline README.md:108]: @MartinKolarik v4 remains the default right? We should mention it here that if not specified we use v4

Comment (MartinKolarik) [reply README.md:108]: Right now yes but I wouldn't want people to rely on it. If we add auto selection later then that would be the default.

Actually now that I think about it, we might have overestimated how hard that would be. Let me come back to that in https://github.com/jsdelivr/globalping/issues/529.

Comment (jimaek) [reply README.md:108]: Ok but until we add auto, we should default to v4 and make sure it's optional to specify the version for better ux

Comment (MartinKolarik) [reply README.md:108]: It is optional to specify it of course, I just wouldn't explicitly document it.

Comment (radulucut) [reply README.md:108]: if `--ipv4` is the default option, do we need to have a flag for it? or am I missing something?

Comment (MartinKolarik) [reply README.md:108]: With the flag the behavior is guaranteed, the default may change.

**Comment by MartinKolarik** [comment]
Probably looks good, I'll review closer and merge later. You can continue with next tasks and target this branch in the next PR.

**Pull request merged**